app/views/application/_favorite.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= link_to "favorite",
+    {controller: 'answers',
```

Comment:
You could probably use a helper here -  flag_favorite_path(answer.id) would work